### PR TITLE
Add ID to persons and events

### LINKED ISSUE #83: Add unique IDs to persons and events

### PR DESCRIPTION
**Pull request by Nicholaskoh1**
Fixes #83

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public AddCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
-        Person person = new Person(name, phone, email, address, tagList);
+        Person person = new Person(name, phone, email, address, tagList, -1);
```
Comment (limyixiang) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:48]: You could consider overloading the constructor method of Person here, such that the ID is initialized to -1 if not specified. This way, your code could be clearer, and the original constructor method signature can still be used.

File: src/main/java/seedu/address/logic/parser/AddEventCommandParser.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public AddEventCommand parse(String args) throws ParseException {
                 argMultiMap.getValue(PREFIX_EVENT_START_DATE).get(),
                 argMultiMap.getValue(PREFIX_EVENT_END_DATE).get());
 
-        Event event = new Event(eventName, eventDescription, eventDuration);
+        Event event = new Event(eventName, eventDescription, eventDuration, -1);
```
Comment (limyixiang) [inline src/main/java/seedu/address/logic/parser/AddEventCommandParser.java:48]: You could consider overloading the constructor method of Event here, such that the ID is initialized to -1 if not specified. This way, your code could be clearer, and the original constructor method signature can still be used.

**Review by limyixiang — COMMENTED** [review]
LGTM! I left some suggestions for you to consider.

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java (hunk 1 above)
Comment (Nicholaskoh1) [reply src/main/java/seedu/address/logic/parser/AddCommandParser.java:48]: Thanks for the feedback! I will change it right now.

File: src/main/java/seedu/address/logic/parser/AddEventCommandParser.java (hunk 2 above)
Comment (Nicholaskoh1) [reply src/main/java/seedu/address/logic/parser/AddEventCommandParser.java:48]: Thanks for the feedback! I will change it right now.

**Pull request merged**